lib/ansible/playbook/included_file.py

```diff
@@ -65,161 +66,147 @@ def process_include_results(results, iterator, loader, variable_manager):
         task_vars_cache = {}
 
         for res in results:
-
```

Comment:
if we are going to refactor this, i would deprecate this function and have the strategy loop over results then on type divide and process includes/normal results differently, right now we seem to be doubling result looping